packages/manager/apps/dedicated/client/app/dedicatedCloud/dedicatedCloud.html

```diff
@@ -7,85 +7,74 @@
     </div>
 
     <div data-ng-if="!loadingInformations">
-        <header class="page-header" role="banner">
-            <div class="row">
-                <div class="col-md-9">
-                    <form name="formEditDisplayName" novalidate>
-                        <div
-                            class="mb-3"
-                            data-ng-if="!dedicatedCloudDescription.editMode"
+        <header class="oui-page-header">
+            <div class="oui-page-header__container d-block">
+                <div class="oui-page-header__content">
+                    <div class="d-flex">
+                        <h1
+                            class="oui-page-header__title"
+                            data-ng-bind="dedicatedCloud.description"
+                        ></h1>
+                        <button
+                            type="button"
+                            class="btn btn-icon ml-2"
```

Comment:
`oui-button` class instead maybe ?

Reply:
Added some nasting padding that I couldn't get rid of unless I create a new stylesheet 😢